File: sample-chat-java/app/src/main/java/com/quickblox/sample/chat/java/utils/qb/callback/QBPushSubscribeListenerImpl.java
package com.quickblox.sample.chat.java.utils.qb.callback;

import android.util.Log;

import com.quickblox.messages.services.QBPushManager;


public class QBPushSubscribeListenerImpl implements QBPushManager.QBSubscribeListener {

    @Override
    public void onSubscriptionCreated() {
        Log.d("Subscription Listener", "Subscription Created");
    }

    @Override
    public void onSubscriptionError(Exception e, int i) {

    }

    @Override
    public void onSubscriptionDeleted(boolean b) {

    }
}

File: sample-chat-kotlin/app/src/main/java/com/quickblox/sample/chat/kotlin/utils/qb/callback/QBPushSubscribeListenerImpl.kt
package com.quickblox.sample.chat.kotlin.utils.qb.callback

import android.util.Log
import com.quickblox.messages.services.QBPushManager


open class QBPushSubscribeListenerImpl : QBPushManager.QBSubscribeListener {

    override fun onSubscriptionCreated() {
        Log.d("Subscription Listener", "Subscription Created")
    }

    override fun onSubscriptionError(e: Exception, i: Int) {

    }

    override fun onSubscriptionDeleted(b: Boolean) {

    }
}